vue-app/src/store/index.ts

```diff
@@ -527,6 +524,19 @@ const getters = {
       (getters.isRoundContributionPhase || getters.isRoundReallocationPhase)
     )
   },
+  hasLowFunds(state: RootState): boolean {
+    const { currentUser, recipientRegistryInfo } = state
+
+    if (
+      currentUser &&
+      currentUser.balance &&
+      recipientRegistryInfo &&
+      recipientRegistryInfo.deposit
```

Comment:
You could use Optional Chaining for this.
```ts
if (currentUser?.balance && recipientRegistryInfo?.deposit) {
  ...
}
```
ref https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining

Reply:
Thanks for suggestion, missed this in cleanup.